src/Elmish.WPF/ViewModel.fs

```diff
@@ -6,69 +6,14 @@ open System.Collections.Generic
 open System.Collections.ObjectModel
 open System.ComponentModel
 open System.Windows
+open Microsoft.Extensions.Logging
 
 open Elmish
 
 
 [<AutoOpen>]
 module internal ViewModelHelpers =
 
-  let historicalMerge
```

Comment:
I just removed this from `master` and rebased `v4`

Reply:
Thanks!